source/support.tex

```diff
@@ -730,9 +730,9 @@
 \begin{itemdescr}
 \pnum
 Measure of the maximum rounding error.\footnote{Rounding error is described in
-ISO/IEC 10967-1 Language independent arithmetic - Part 1
-Section 5.2.8 and
-Annex A Rationale Section A.5.2.8 - Rounding constants.}
+LIA-1
+Section 5.2.4 and
+Annex C Rationale Section C.5.2.4 --- Rounding constants.}
```

Comment:
The name of this section is "Rounding and rounding constants", not merely "Rounding constants".